data/org.freedesktop.portal.InputCapture.xml

```diff
@@ -106,14 +117,46 @@
           It is best to view this set as a negative confirmation - a
           capability that was requested but is missing is an indication that
           this application may not capture events of that capability.
+
+        This method was added in version 2 of this interface.
     -->
-    <method name="CreateSession">
+    <method name="InitSession">
       <arg type="s" name="parent_window" direction="in"/>
       <annotation name="org.qtproject.QtDBus.QtTypeName.In1" value="QVariantMap"/>
       <arg type="a{sv}" name="options" direction="in"/>
       <arg type="o" name="handle" direction="out"/>
     </method>
 
+    <!--
+        CreateSession:
+        @handle: Object path for the :ref:`org.freedesktop.impl.portal.Request` object representing this call
+        @session_handle: Object path for the :ref:`org.freedesktop.impl.portal.Session` object representing the session being created
+        @app_id: App id of the application
+        @parent_window: Identifier for the application window, see :doc:`window-identifiers`
+        @options: Vardict with optional further information
+        @response: Numeric response
+        @results: Vardict with the results of the call
+
+        Create a capture input session.
+
+        Supported keys in the @options vardict include:
+
+        * ``capabilities`` (``u``)
+
+          Bitmask of requested capabilities, see the
+          SupportedCapabilities property. This value is required.
+
+        The following results get returned in the @results vardict:
+
+        * ``session_id`` (``s``)
+
+          The session id. A string representing the created capture input session.
+
+        * ``capabilities`` (``u``)
+
+          The capabilities available to this session. This is always a
+          subset of the requested capabilities.
+    -->
```

Comment:
Removing the `<method name="CreateSession">` is an API break, we need to keep that one around as-is.

Reply:
I replaced the method in both XML files.